Runtimes/Core/runtime/CMakeLists.txt

```diff
@@ -86,7 +86,7 @@ endif()
 target_compile_definitions(swiftRuntime
   PRIVATE
     -DSWIFT_RUNTIME
-    -DSWIFT_TARGET_LIBRARY_NAME=swiftRuntime
+    -DSWIFT_TARGET_LIBRARY_NAME=swiftRuntimeCore
```

Comment:
This is incorrect, this should be `swiftCore`. The way that this macro is used is with the CPP macro programming in `Visibility.h`, and the only valid options are `swiftCore`, `swift_Concurrency`, `swiftDistributed, `swift_Differentiation`. The value is used to select if the symbol is in the current module's ABI or not.